packages/jazz-browser/src/storageOptions.ts

```diff
@@ -17,7 +17,7 @@ export function getStorageOptions(storage?: StorageConfig): {
     !storage ||
     (Array.isArray(storage) && storage.includes("indexedDB")) ||
     storage === "indexedDB" ||
-    !useSingleTabOPFS;
+    (!useSingleTabOPFS && storage !== "disabled");
```

Comment:
Added the disabled option to storage to be able to test the sync without a shared storage between accounts.